### PR TITLE
add event.Value type check

### PR DESCRIPTION
**Pull request by daqingshu**

### CODE REVIEW

**Comment by tianxiaoliang** [comment]
Can one of the admins verify this patch?

File: eventlistener/lager_event_listener.go [hunk 1]
```diff
@@ -24,14 +24,18 @@ type LagerEventListener struct {
 //Event is a method for Lager event listening
 func (e *LagerEventListener) Event(event *core.Event) {
 	logger := openlogging.GetLogger()
-	logger.Debugf("Get lager event, key: %s, type: %s", event.Key, event.EventType)
+	fmt.Printf("Get lager event, key: %s, type: %s, value: %v", event.Key, event.EventType, event.Value)
```
Comment (tianxiaoliang) [inline eventlistener/lager_event_listener.go]: merge 26 and 27 into one line, and from now must use structured log info, refer to my code "openlogging.WithTag"

**Comment by tianxiaoliang** [comment]
refer to

https://github.com/apache/servicecomb-kie/blob/4d11050f5090cf229d6143505f51b32dfd57e367/client/client.go#L99

**Comment by tianxiaoliang** [comment]
you can change log level to info then， but never use fmt to print

**Comment by daqingshu** [comment]
> refer to
> 
> https://github.com/apache/servicecomb-kie/blob/4d11050f5090cf229d6143505f51b32dfd57e367/client/client.go#L99

``` golang

```

> you can change log level to info then， but never use fmt to print

ok, plz review again.

File: eventlistener/lager_event_listener.go (hunk 1 above)
Comment (daqingshu) [reply eventlistener/lager_event_listener.go]: change.

**Comment by daqingshu** [comment]
fix #633

**Pull request merged**